doc/policy_tutorial/POLICY_RUN_LOCALLY.md

```diff
@@ -124,7 +135,7 @@ if secret_header ~= self.secret then
 The policy is now executing. You must provide the secret header in the request to pass the validation. Issue the following HTTP request:
 
 ```shell
-$ curl localhost:8080 -H 'secret: mysecret'
+$ curl "http://localhost:8080/?user_key=" -H 'secret: mysecret' -H "Host: one"
 GET / HTTP/1.1
 X-Real-IP: 127.0.0.1
 Host: echo
```

Comment:
This is the response from echo server? If so then the `Host` header should be `echo-api.3scale.net` right?

Reply:
hmm actual response
```bash
curl -i -H "secret: mysecret" -H "Host: one" "http://${APICAST_IP}:8080/?user_key="

HTTP/1.1 200 OK
Server: openresty
Date: Tue, 24 Sep 2024 11:13:29 GMT
Content-Type: text/plain
Transfer-Encoding: chunked
Connection: keep-alive

GET /?user_key= HTTP/1.1
X-Real-IP: 172.22.0.1
Host: echo:8081
User-Agent: curl/8.2.1
Accept: */*
secret: mysecret
```
so not `echo-api.3scale.net` though not sure why . I will update accordingly